lib/Controller/ConfigController.php

```diff
@@ -231,12 +247,22 @@ public function oauthRedirect(string $code = '', string $state = ''): RedirectRe
 		);
 	}
 
-	private function storeUserInfo(GitlabAccount $account): void {
+	/**
+	 * @param GitlabAccount $account
```

Comment:
```suggestion
```
Not necessary as the type is already specified

Reply:
it was just the autocomplete, doesn't harm though, right?

Reply:
Not really, but it's cleaner to skip it as otherwise the two type definitions could drift apart.

Reply:
psalm would warn us if we do that. Also, it seems better to just mention all of them instead of selective params, no? Could be better for some IDEs too.